ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -713,9 +704,8 @@ const ControlledTierCard = (props: {profile: Profile, reload: Function, spinnerP
                           enabled={isEligible}
                           text={`${institutionDisplayName} must allow you to access ${ctDisplayName} data`}/>
       {displayEraCommons &&
-         <ControlledTierEraModule profile={profile} eligible={isEligible} spinnerProps={spinnerProps}/>}
-      {unsafeAllowSelfBypass &&
-        <SelfBypassCTComplianceTrainingModule profile={profile} reload={reload} eligible={isEligible} spinnerProps={spinnerProps}/>}
+       <ControlledTierEraModule profile={profile} eligible={isEligible} spinnerProps={spinnerProps}/>}
```

Comment:
eRA commons here seems like a complicating factor - probably there is room for some cleanup here. I considered including some refactoring here, but the logic was a bit tricky to follow.

Let me know if you have a preference on how I handle it in this PR.

Reply:
let's leave it alone for now